.gitignore

```diff
@@ -4,3 +4,5 @@
 .phpunit.result.cache
 composer.lock
 infection.log
+
+.idea
```

Comment:
This should be in your global .gitignore.

Reply:
Okay, fixed!